### PR TITLE
Barge Update

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
Updated Barge:
- Added waste loop, mapped emergency lights.

## Why / Balance
To curb CO2 build up.

## Technical details
.yml changes

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame.

**Changelog**
:cl: erhardsteinhauer
- tweak: Tweaked Barge - added waste loop to atmos.

### CODE REVIEW

**Comment by TsjipTsjip** [comment]
YML-wise, this PR is too complex to review by hand. I'll load this in-game in a bit.

File: Resources/Maps/Shuttles/mail.yml
Comment (TsjipTsjip) [inline Resources/Maps/Shuttles/mail.yml:1]: This is a commit from a different PR and should be removed from this one.

File: Resources/Maps/Shuttles/barge.yml
Comment (TsjipTsjip) [inline Resources/Maps/Shuttles/barge.yml:1]: This scrubber should be moved away from the wast piping itself for clarity, analogous to the vent in the same room.

image

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
Couple of things, the rest looks good to go.

File: Resources/Maps/Shuttles/mail.yml
Comment (ErhardSteinhauer) [reply Resources/Maps/Shuttles/mail.yml:1]: idk how to "re-base" (or "re-master", hell, I don't even know the proper term for this)  the commit. Can make new one.

File: Resources/Maps/Shuttles/barge.yml
Comment (ErhardSteinhauer) [reply Resources/Maps/Shuttles/barge.yml:1]: Aye.

Comment (ErhardSteinhauer) [reply Resources/Maps/Shuttles/barge.yml:1]: image

File: Resources/Maps/Shuttles/mail.yml
Comment (ErhardSteinhauer) [reply Resources/Maps/Shuttles/mail.yml:1]: used google, re-based this branch to my master

**Review by MagnusCrowe — APPROVED** [review]

**Pull request merged**